src/Worker/UpdateContact.php

```diff
@@ -38,6 +39,11 @@ class UpdateContact
 	 */
 	public static function execute(int $contact_id)
 	{
+		// Silently dropping the task if the contact is blocked
+		if (Contact::isBlocked($contact_id)) {
```

Comment:
Please also check if the contact's server could be blocked as well.

Reply:
It is already part of `Contact::isBlocked`.